rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,221 @@
+
+(See *Refinement* in [*Scrum Guide*](https://scrumguides.org/docs/scrumguide/v2017/2017-Scrum-Guide-US.pdf) and [*Nexus Guide*](http://scrumorg-website-prod.s3.amazonaws.com/drupal/2018-01/2018-Nexus-Guide-English_0.pdf) for guidance.)
+
+#### Refinement and ZenHub Pipelines
+
+New items in the *dcp* repo appear in the ZenHub *New* pipeline and MUST be triaged into the *Product Backlog* pipeline, the *Icebox* pipeline, or the *Closed* pipeline.
+
+The ZenHub *Product Backlog* pipeline is the top-level DCP product backlog which is ordered during Refinement.
+
+The ZenHub *Icebox* pipeline SHOULD be used with great constraint in the *dcp* repo. It's not a pipeline for avoiding hard decisions during Refinement. It's intended to document potential *futures* from the DCP roadmap. Individual component repo(s) MAY use the *Icebox* pipeline in a manner of their own choosing.
+
+No issues in the *dcp* repo MUST be assigned to the *Epic* pipeline. Individual component repo(s) MAY use the *Epic* pipeline in a manner of their own choosing.
+
+### Sprint Planning
+
+Prior to the start of the DCP Sprint, component teams MUST identify their *Sprint Backlog* - a slice of items from the the ZenHub *Product Backlog* pipeline in their repo which are assigned to the ZenHub Milestone for the Sprint and moved to the ZenHub *Sprint Backlog* pipeline.
+
+The number of items selected is solely up to the Development Team. 
+- [ZenHub story points](https://help.zenhub.com/support/solutions/articles/43000034282) MUST be assigned to all *Sprint Backlog* items. (Story points are required to enable [ZenHub burndown charts](https://help.zenhub.com/support/solutions/articles/43000010356-track-sprint-progress-with-burndown-charts) to track the progress of a Sprint.)
+  - [How to start?](https://www.scrum.org/forum/scrum-forum/6774/how-decide-how-many-story-points-take-first-sprint)
+- Research and prototyping tasks SHOULD be modeled as Spikes.
+- Dependencies between components within a Sprint MUST be communicated between the respective teams and surfaced using [ZenHub Dependencies](https://help.zenhub.com/support/solutions/articles/43000010349). 
+
+At the start of the Sprint, the single DCP Sprint Backlog SHOULD be reviewed by the **PM Team** and **Architecture Team** in their oversight roles.
+
+### Sprint(ing)
+
+During Development, the Development Team assigns issues from the ZenHub *Sprint Backlog* pipeline and move to the *In Progress* pipeline. 
+
+When a pull request is filed and in review for an *In Progress* item, the issue is moved to the *Review* pipeline. 
+
+If the pull request is rejected, the issue returns to *In Progress*.
+
+Otherwise, the pipeline tracks the issue through the DCP [promotion sequence](https://allspark.dev.data.humancellatlas.org/dcp-ops/docs/wikis/SOP:%20Releasing%20new%20Versions%20of%20DCP%20Software).
+
+The approved pull request is merged to the default branch, deployed to Development, and the issue moves to the *Development* pipeline.
```

Comment:
… collides with its use here. No one calls the `dev` deployment `Development`. I don't have a solution other than using the names I originally proposed: `Merged`, `Integrated`, `Staged` and `Done`. I still prefer those.

Reply:
I understand the urge to name the added pipelines after the deployments, but we have been using inconsistent terminology for those, e.g. `prod` vs `production`.  I find it more sound to name the pipelines by the intent *behind* deployment. The intent behind the `integration` deployment is to integrate changes with other components. The intent behind the staging deployment is to stage changes for the final push to production.